### ISSUE TITLE
[BUG] WSmatrix not working correctly after suspend

### ISSUE THREAD

**Issue by amrayach** [mzur/gnome-shell-wsmatrix#170]
Hello, 
I want to thanks everyone for contributing to this awesome project, I haven't had any OS without this extension.

I am facing the problem that after closing my lid or doing a suspend WSmatrix doesn't work properly after and my desktop environment is messed up,  bars change positions, no more apps are displayed anymore and if I click the super key the system hangs until I log out and log in again after doing that I lose my session and all my opened windows.

here is the relevant part of the journalctl /usr/bin/gnome-shell output:

```
Sep 09 14:26:03 ammer-T14s gnome-shell[1800]: JS ERROR: Could not load extension gnome-shell-wsmatrix-gnome-40: Error: Missing metadata.json
Sep 09 14:26:03 ammer-T14s gnome-shell[1800]: Extension wsmatrix@martin.zurowietz.de already installed in /home/ammer/.local/share/gnome-shell/extensions/wsmatrix@martin.zurowietz.de. /usr/share/gnome-shell/extensions/wsmatrix@martin.zurowietz.de will not be loaded
Sep 09 14:31:04 ammer-T14s gnome-shell[1800]: JS ERROR: Extension wsmatrix@martin.zurowietz.de: TypeError: Main.overview._overview.controls is undefined
                                              WorkspacesView@/home/ammer/.local/share/gnome-shell/extensions/wsmatrix@martin.zurowietz.de/overview/workspacesView.js:13:9
                                              OverviewManager@/home/ammer/.local/share/gnome-shell/extensions/wsmatrix@martin.zurowietz.de/overview/overviewManager.js:16:40
                                              enable@/home/ammer/.local/share/gnome-shell/extensions/wsmatrix@martin.zurowietz.de/extension.js:11:33
Sep 09 14:31:06 ammer-T14s gnome-shell[1800]: [pixel-saver]: Can't find original state for mzur/gnome-shell-wsmatrix at gnome-40 — Mozilla Firefox with id 0xc00003
```

Here is the complete log:
[ws_log.txt](https://github.com/mzur/gnome-shell-wsmatrix/files/7136560/ws_log.txt)

hopefully, this problem could be addressed when your time allows it.

Thanks in advance and Best regards, 
Ammer Ayach

**Comment by ebeem** [comment]
Can you please confirm which GNOME version are you using?
Is it `GNOME 40`?

**Comment by amrayach** [comment]
Yes indeed, I am using Manjaro GNOME 40 build currently.

Sorry, I thought I was posting this issue under the GNOME 40 branch that's why I skipped mentioning that.


Best,

AA

**Comment by ebeem** [comment]
No problem, this was reported in the PR already.
Let's see what we can do about it.

Many thanks for taking the time to report this issue and providing proper logs.

**Comment by mzur** [comment]
I think the two most recent commits to the gnome-40 branch should fix the errors you see in the log ( 9a4da11feab99f0811b93ab136e652ab63f87cb9, 2c218a6082ca1c24af47d607310e0fb003ed16bd ). Please report back if you see more errors.
